README.md

```diff
@@ -1,9 +1,100 @@
+<img alt="asteroidos logo" width="190" src="https://asteroidos.org/public/img/logo.png">
+</a>
+</div>
+
+  <p align="center">
+    Synchronize the time, notifications and more with your AsteroidOS smartwatch.
+    <br />
+    <br />
+    <a href = "https://hosted.weblate.org/projects/asteroidos/sync/">
+        <strong>Help us translate AsteroidOS Sync via Weblate </strong>
+    </a>
+    <br />
+    <br />
+        <a href="https://asteroidos.org/wiki/documentation/"><strong>Explore the docs </strong></a>
+    <br />
+    <a href="https://github.com/AsteroidOS/AsteroidOSSync/issues">Report Bug</a>
+    .
+    <a href="https://github.com/AsteroidOS/AsteroidOSSync/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement">Request Feature</a>
+    <br />
+    <a href="https://f-droid.org/packages/org.asteroidos.sync/" target="_blank">
+    <img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png" alt="Get it on F-Droid" height="90"/></a>
+   </p>
+</p>
+
+# Table of Contents 
+* [About The Project](#about-the-project) 
+* [Getting Started](#getting-started)
+  * [Development Instructions](#development-instructions) 
+* [Roadmap](#roadmap)
+* [Contributing](#contributing)
+  * [IRC](#IRC)
+
+# About The Project 
+
+AsteroidOS is an open-source operating system for smartwatches.
+
+### Freedom & Privacy
+
+We believe that when it comes to wearable devices, users should have full control over their machines and data. AsteroidOS has been created from the ground-up with modularity and freedom in mind. For this reason, it is a free and open-source project.
+
+### Ready for your wrist
+
+AsteroidOS unleashes the potential of your watch with up to 48 hours of autonomy and a set of apps including everything you need on a smartwatch: an agenda, an alarm clock, a calculator, a music controller, settings, a stopwatch, a timer and a weather forecast app.
+
+### Proven building blocks
+
+AsteroidOS is built upon a rock-solid base system. Qt and QML are used for fast and easy app development. OpenEmbedded provides a full GNU/Linux distribution and libhybris allows easy porting to most Android and Wear OS watches.
+
+### Community-friendly
+
+Since its creation in 2015, AsteroidOS has been led by Florent Revest, but is open towards external contribution and collaboration. The project is still in active development and if you want to help, make sure to check the Get Involved page.
+
+# Getting Started 
+
+AsteroidOSSync can be downloaded from f-droid [here](https://f-droid.org/packages/org.asteroidos.sync/) 
+
+If you don't have a device running AsteroidOS, instructions for installation can be found [here](https://asteroidos.org/install/) 
+
+### Development Instructions 
+To get started with a developing for AsteroidOSSync, fork and clone the project, and import into any Android IDE. 
+
+# Roadmap 
+See the [open issues](https://github.com/AsteroidOS/AsteroidOSSync/issues) for a list of proposed features and known issues 
+
+# Contibruting
```

Comment:
"Contibruting": Can you please fix this typo and clean up the commit history.

Reply:
ah! fixed!